desktop/apps/dev/index.coffee

```diff
@@ -16,3 +16,6 @@ app.get '/dev/blank', (req, res) ->
   res.render 'blank',
     env: NODE_ENV
     view_cache_enabled: app.enabled 'view cache'
+
+app.get '/yeg3dqrlq548zc77ggvfipouil1l1e.html', (req, res) ->
```

Comment:
Strange verification process—I wonder if this should be stored in a config variable in case it needs to change in the future?

Reply:
reminds me of the Google Analytics verification tool 👍